### PR TITLE
Lt iv1.2

### PR DESCRIPTION
**Pull request by urashidmir**
LTI handler added.

### CODE REVIEW

**Comment by urashidmir** [comment]
LTI Handler added

File: components/Login/LTI.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+class LTI extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.componentDidMount = this.componentDidMount.bind(this);
+        this.state = {
+            queryData: this.props.currentRoute.query.data
+        };
+        cookie.save('user_json_storage', this.state.queryData, {path: '/'});
```
Comment (TBoonX) [inline components/Login/LTI.js]: Please also set maxAge or expires. See https://github.com/slidewiki/slidewiki-platform/blob/master/plugins/UserStorage/userStoragePlugin.js#L99

File: components/Login/LTI.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    componentWillMount() {
+        console.log('Will be called on the server...');
+    }
+    componentDidMount() {
+        console.log('LTI. componentDidMount');
+    }
+
+    render() {
+        //console.log('LTILogin.render called. queryData='+this.state.queryData);
+        console.log('LTILogin.render called.');
```
Comment (TBoonX) [inline components/Login/LTI.js]: Remove/outcomment the log

File: components/Login/LTI.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        super(props);
+        this.componentDidMount = this.componentDidMount.bind(this);
+        this.state = {
+            queryData: this.props.currentRoute.query.data
+        };
+        cookie.save('user_json_storage', this.state.queryData, {path: '/'});
+    }
+
+    componentWillMount() {
+        console.log('Will be called on the server...');
```
Comment (TBoonX) [inline components/Login/LTI.js]: Remove/outcomment the log

File: components/Login/LTI.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    }
+    componentDidMount() {
+        console.log('LTI. componentDidMount');
```
Comment (TBoonX) [inline components/Login/LTI.js]: Remove/outcomment the log

File: components/Login/LTI.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {handleRoute} from 'fluxible-router';
+import {connectToStores} from 'fluxible-addons-react';
+import {navigateAction} from 'fluxible-router';
+import ReactDOM from 'react-dom';
+import cookie from 'react-cookie';
+let classNames = require('classnames');
+
+const NAME = 'ltilogin_data';
```
Comment (TBoonX) [inline components/Login/LTI.js]: Could be removed.

File: components/Login/LTI.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+let queryData = '';
```
Comment (TBoonX) [inline components/Login/LTI.js]: Could be removed.

**Comment by kprist** [comment]
@urashidmir if we don't need this, please close this PR.

**Pull request closed without merging**